frontend/src/components/partners/partnersActivity.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import { OHSOME_STATS_BASE_URL } from '../../config';
+import { FormattedMessage } from 'react-intl';
+
+import ReactPlaceholder from 'react-placeholder';
+import PartnersProgresBar from './partnersProgresBar';
+import messages from './messages';
+
+export const Activity = (partner) => {
+  const [data, setData] = useState(null);
+
+  const fetchData = async () => {
+    try {
+      let primaryHashtag = partner.primary_hashtag.trim();
+      if (primaryHashtag.startsWith('#')) {
+        primaryHashtag = primaryHashtag.slice(1);
+      }
+      primaryHashtag = primaryHashtag.toLowerCase();
+
+      const secondaryHashtags = partner.secondary_hashtag
+        .split(',')
+        .map((tag) => tag.trim().replace('#', '').toLowerCase())
+        .join(',');
+      const response = await fetch(
+        OHSOME_STATS_BASE_URL + `/hashtags/${primaryHashtag},${secondaryHashtags}`,
```

Comment:
@CristianAmici This component is failing to retrieve stats, I think this is because there's an issue with the URL here.

Please add `/stats`:

`OHSOME_STATS_BASE_URL + `/stats/hashtags/${primaryHashtag},${secondaryHashtags}`

Like you did in `views/partnerStats`

CC @dgquail